resources/assets/js/components/User.js

```diff
@@ -33,21 +33,21 @@ export default class User extends Component {
 
                         <div className="entry">
                             <p className="title">Website</p>
-                            <p className="value"><a href={this.state.user.website} target="_blank">{this.state.user.website}</a></p>
+                            <p className="value"><a href={`//${this.state.user.website}`} target="_blank">{this.state.user.website}</a></p>
```

Comment:
I am not sure what exactly your intention is, but this wouldn't do what you think it would do.

Reply:
Uhm, let's start from here: What do you think i want it to do ¿